bigbluebutton-html5/imports/ui/components/audio/audio-controls/styles.scss

```diff
@@ -31,8 +31,32 @@
   }
 
   &.glow {
-    span:first-child{
-      box-shadow: 0 0 0 1px #ffffff, 0 0 0 1px;
-    }
+    -webkit-animation: pulse 1s infinite ease-in-out;
+    animation: pulse 1s infinite ease-in-out;
+    border-radius: 4em;
```

Comment:
there is a border-radius variable in general.sccs that should be used here

Reply:
The border-radius variable in the general.scss(.2rem) is not the correct value here for this situation.